docs/source/futures.rst

```diff
@@ -52,7 +52,7 @@ Then you can submit individual tasks using the ``submit`` method.
    a = client.submit(inc, 10)  # calls inc(10) in background thread or process
    b = client.submit(inc, 20)  # calls inc(20) in background thread or process
 
-Submit returns a ``Future``, which refers to a remote result.  This result may
+``submit`` returns a ``Future``, which refers to a remote result.  This result may
```

Comment:
I have a slight preference to not start sentences with lower-cased code-highlighted words.  Maybe the following change?

```suggestion
The ``submit`` function returns a ``Future``, which refers to a remote result.  This result may
```

Reply:
Tthis looks better indeed.